docs/deployments/overview.mdx

```diff
@@ -29,7 +29,7 @@ In most cases, you will want to clone your development instance. This will copy
 
 ## API keys and environment variables
 
-A common mistake when deploying to production is forgetting to change your API keys to your production instances keys. The best way to set this up is to make use of environment variables. All modern hosting providers, such as AWS, GCP, Vercel, Heroku, and Render, make it easy to add these values. Locally, you should use a `.env` file. This way, these values are being set dynamically depending on your environment. Here's a list of Clerk variables you'll need to change:
+A common mistake when deploying to production is forgetting to change your API keys to your production instances keys. The best way to set this up is to make use of environment variables. All modern hosting providers, such as AWS, GCP, Vercel, Heroku, and Render, make it easy to add these values. Locally, you should use an `.env` file. This way, these values are being set dynamically depending on your environment. Here's a list of Clerk variables you'll need to change:
```

Comment:
are you sure this is right? i would have gone for the previous version, but maybe im bad at grammar idk

Reply:
like, if you remove `.env` and say just "use a file" vs "use an file", the former feels more accurate, and `.env` is basically an adjective in this instance so it shouldn't make an impact right

Reply:
@LekoArts had a debate on this - and we chose to go with "an" as if someone is saying "an env file" instead of a "a dotenv file". at the end of the day, its preference, and we had to stick with one